recipes/xtensor-blas/all/conanfile.py

```diff
@@ -0,0 +1,92 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import textwrap
+
+required_conan_version = ">=1.43.0"
+
+
+class XtensorBlasConan(ConanFile):
+    name = "xtensor-blas"
+    description = "Extension to the xtensor library, offering bindings to BLAS and LAPACK libraries through cxxblas and cxxlapack from the FLENS project."
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xtensor-stack/xtensor-blas"
+    topics = ("numpy", "multidimensional-arrays",
+              "tensors", "openblas", "blas", "lapack")
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        if tools.Version(self.version) == "0.17.2":
+            self.requires("xtensor/0.21.4")
+        else:
+            self.requires("xtensor/0.24.0")
+        self.requires("openblas/0.3.17")
```

Comment:
```suggestion
        self.requires("openblas/0.3.20")
```
Please, try the latest version

Reply:
package test still failed if I built openbls with `build_lapack=False`.

Reply:
Need to simplify the test, check only the project version or call a version simple method. `build_lapack=True` is not available.